packages/embeds/embed-core/playwright/config/playwright.config.ts

```diff
@@ -150,6 +150,8 @@ expect.extend({
       }, 500);
     });
 
+    clearInterval(iframeReadyCheckInterval);
```

Comment:
It was causing `getActionFiredDetails` to be unnecessarily called in timer when it wasn't needed.